### DISCUSSION TITLE
Minify Output and Elliptical Arc

### DISCUSSION THREAD

**Discussion by njsokalski** [Yqnn/svg-path-editor#85] [General]
I would like to start by saying that this is one of the best online tools I have ever found, so this is by no means meant to be an insult, just something I want to mention. Although I haven't yet examined the official specifications for SVG, I have noticed that when including an Elliptical Arc (the A) and enabling the Minify Output option, the large-arc-flag & sweep-flag get crunched against the x value. The SvgPathEditor does not seem to have a problem with this, but when using the code as path data in a Xamarin.Android app, it causes an error. It took me a while to realize that this was the reason, and I would like to be able to use the Minify Output feature without needing to manually add extra spaces. Hopefully this message can also help anybody else who has been receiving this error. Thank you again for this wonderful tool!

**Comment by Yqnn** [comment]
Thanks for the report 👍 

From SVG specifications, the space between numbers and flags is optional : https://www.w3.org/TR/SVG11/paths.html#PathDataBNF , it's a "standard" optimization to remove it.

So I guess this is a bug of Xamarin.
I may add extra parameters in the future, but for now disabling minification is the only workaround.
